biorbd_optim/path_conditions.py

```diff
@@ -137,8 +137,9 @@ def check_and_adjust_dimensions(self, nb_elements, nb_shooting, element_type):
                     f"Invalid number of column for InterpolationType.EACH_FRAME (ncols = {self.shape[1]}), "
                     f"the expected number of column is {self.nb_shooting}"
                 )
-        else:
-            raise RuntimeError(f"InterpolationType is not implemented yet")
+        # Intended ???
+        # else:
+        #     raise RuntimeError(f"InterpolationType is not implemented yet")
```

Comment:
I don't remember if it is part of the PR I made you. If not, please remove these 3 lines, they are not in the master branch anymore and has, for some unknown reason, reappeared in your branch